fixups/RegLegacyFixups/readme.md

```diff
@@ -65,17 +69,51 @@ The following Windows API calls are supported for this fixup type.
 ### Configuration for FakeDelete
 When the `type` is specified as `FakeDelete`, the `remediation` element is an array of remediations with a structure shown here:
 
+| Tag | Purpose |
+| --- | ------- |
 | `hive` | Specifies the registry hive targeted. |
 | `patterns` | An array of regex strings. The pattern will be used against the path of the registry key being processed. |
 
 The value of the `hive` element may specified as shown in this table:
 
 | Value | Purpose |
+| ----- | ------- |
 | HKCU  | HKEY_CURRENT_USER |
 | HKLM  | HKEY_LOCAL_MACHINE |
 
 The value for the element `patterns` is a regex pattern string that specifies the registry paths to be affected.  This regex string is applied against the path of the key/subkey combination without the hive.
 
+## DeletionMarker Remediation Type
+The following Windows API calls are supported for this fixup type. 
+
+> * RegGetValue
+> * RegQueryValueEx
+> * RegQueryMultipleValues
+> * RegQueryInfoKey
+> * RegEnumValue
+> * RegEnumKeyEx
+> * RegOpenKeyEx
+> * RegOpenKeyTransacted
+
+### Configuration for DeletionMarker
+When the `type` is specified as `DeletionMarker`, the `remediation` element is an array of remediations with a structure shown here:
+| Tag | Purpose |
+| --- | ------- |
+| `hive` | Specifies the registry hive targeted. |
+| `key` | Contains a regex string. The key will be used against the path of the registry key being processed. |
+| `values` | Contains an array of regex strings. The value will be used against the name of the registry value being processed in the key specified. This is an optional field. Omitting this field would hide the key specified. |
```

Comment:
It looks values is optional from the example. Please mention it is optional and what is default value considered. Please refer to [this](https://github.com/microsoft/MSIX-PackageSupportFramework/tree/develop/PsfLauncher#json-schema)